### DISCUSSION TITLE
Define encoding in redshift

### DISCUSSION THREAD

**Discussion by night-fury-web** [dbt-labs/docs.getdbt.com#1834] [Archive]
I have my source table with columns in zstd encoding.

After running dbt, the output column is using az64 encoding.

How can I keep zstd encoding after dbt run?

**Comment by joellabes** [comment]
@night-fury-web this isn't possible in dbt, have a look at https://github.com/dbt-labs/dbt-core/issues/1016 for a deeper dive into why. In short, Redshift doesn't support specifying column encoding in a `create table as select` statement which is how dbt builds all its models.
